### PR TITLE
[Tools] Enable PHPCS for CouchDB_Confirm_Integrity.php

### PR DESCRIPTION
**Pull request by laemtl**
Replaces #6002

### CODE REVIEW

**Comment by CamilleBeau** [comment]
This is failing travis because short array syntax isn't being used

**Comment by laemtl** [comment]
@CamilleBeau Solved!

File: tools/CouchDB_Confirm_Integrity.php [hunk 1]
```diff
@@ -88,22 +88,24 @@ function run()
...
+                ]
             );
 
             if (!empty($sqlDB) && $sqlDB['cActive'] == 'N') {
                 print "PSCID $pscid is inactive but $row[id] still exists. 
                 Deleting Doc.\n";
 
                 $this->CouchDB->deleteDoc($row['id']);
-            } else if (!empty($sqlDB) && $sqlDB['Active'] != 'Y') {
+            } elseif (!empty($sqlDB) && $sqlDB['Active'] != 'Y') {
```
Comment (driusan) [inline tools/CouchDB_Confirm_Integrity.php:102]: What rule was complaining about this?

Comment (laemtl) [reply tools/CouchDB_Confirm_Integrity.php]: @driusan Nothing seems to justify this change. I reverted it.

**Pull request merged**